dom.bs

```diff
@@ -3031,15 +3031,54 @@ standards that want to define APIs shared between <a for=/>documents</a> and
 <h4 id=interface-parentnode>Mixin {{ParentNode}}</h4>
 
 <p>To <dfn export lt="converting nodes into a node">convert nodes into a node</dfn>, given
-<var>nodes</var> and <var>document</var>, run these steps:
+<var>parent</var>, <var>nodes</var> and <var>document</var>, run these steps:
 
 <ol>
+ <li><p>Let <var>isScriptElement</var> be <var>parent</var> is a {{HTMLScriptElement}}.
+ <li><p>For each <var>value</var> in <var>nodes</var>:
+  <ol>
+   <li>If <var>value</var> is a <a for=/>node</a>, then:
+    <ol>
+     <li>If <var>isScriptElement</var> is false, skip to the next <var>value</var>.
```

Comment:
then continue* (see also Infra)

Reply:
You resolved this, but it doesn't appear to be resolved.

Reply:
Apologies idk what happened there, I addressed it below this line must have missed this. Will do that now